sdk/iot/hub/samples/CMakeLists.txt

```diff
@@ -86,6 +86,30 @@ target_link_libraries(
   OpenSSL::Crypto
   )
 
+# Telemetry Sample
+add_executable (
+  paho_iot_hub_sas_telemetry_example
+  ${CMAKE_CURRENT_LIST_DIR}/src/paho_iot_hub_sas_telemetry_example.c
+  ${CMAKE_CURRENT_LIST_DIR}/src/sample_base64.c
+  )
+
+target_link_libraries(
+  paho_iot_hub_sas_telemetry_example PRIVATE
+  az_iot_hub
+  az_iot_common
+  az_core
+  )
+
+# PAL
```

Comment:
is this really a PAL? It's just a platform

Reply:
platform vs (platform abstraction layer). I followed what the other `CMakeLists.txt` do here